api/test/unit/validation/schemaValidator.test.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+var expect = require("chai").expect;
```

Comment:
This doesn't really test anything since it is more a recoding of the schema.
We should be testing the schema validation not reaching through to see how it is defined.